app/build.gradle

```diff
@@ -77,7 +77,6 @@ android {
             signingConfig signingConfigs.debug
             applicationIdSuffix '.debug'
             versionNameSuffix "-debug"
-            shrinkResources true
```

Comment:
I think it's good. If you can check apk size, I'm appreciated. Do you have a time?